test/svg-test-helper.js

```diff
@@ -80,3 +82,88 @@ export const calculateD3Path = (props, pathType, index = 0) => {
 
   return undefined;
 };
+
+export const parseSvgPathCommands = (commandStr) => {
+  const matches = commandStr.match(
+    /[MmLlHhVvCcSsQqTtAaZz]+[^MmLlHhVvCcSsQqTtAaZz]*/g
+  );
+
+  return matches.map((match) => {
+    const name = match.charAt(0);
+    const args = match
+      .substring(1)
+      .split(",")
+      .map((arg) => {
+        return parseFloat(arg, 10);
+      });
+
+    return {
+      raw: match,
+      name,
+      args
+    };
+  });
+};
+
+export const getPathCommandsFromContainer = (container) => {
+  const commandStr = container.querySelector("path").getAttribute("d");
+  return parseSvgPathCommands(commandStr);
+};
+
+const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
+  const commands = parseSvgPathCommands(commandString);
+  return commands.every(
+    (command, index) => command.name === shapeSeqeuence[index]
+  );
+};
+
+export const svgExpectations = {
```

Comment:
We should update this syntax a bit and export one function at a time rather than exporting an object with multiple functions. I'm not sure if something like tree shaking matters in test utils, but we have been updating the rest of our utils to follow this convention. 

If we want them to be scoped, we can just create a different file that exports each function individually, and then we could do something like `import * as svgHelpers from '../svg-helpers'`.